src/service/product-service.ts

```diff
@@ -326,12 +362,17 @@ export default class ProductService {
       category: true,
     };
 
-    const userFilter: any = {};
+    if (params.returnContainers) relations.containers = {
```

Comment:
Do products even have a link to containers?

Reply:
I created it. It was missing, but adding it made the implementation of the `product-service`, `container-service` and `point-of-sale-service` more consistent.